src/ua/griddynamics/Game.java

```diff
@@ -0,0 +1,183 @@
+package ua.griddynamics;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Game {
+    static final int WIDTH = 9;
+    static final int HEIGHT = 9;
+
+    public final char X = 'X';
+    public final char FREE = '/';
+    public final char MINE = '*';
+    public final char POINT = '.';
+
+    private final char[][] hiddenField;
+    private final char[][] visibleField;
```

Comment:
Replace `hiddenField` and `visibleField` by just one array Cell[][] cells;
Where Cell is a custom class, that contains all needed information